magento/magento2
# Fixes #11022 AttributeRepository::getList searchCriteria Filtergroups not included

### Description

The Filter Groups and the Sort Order of the $searchCriteria parameter have not been included in the searchCriteria Object that is finally provided for the eav set repository.

This Pull request sets the filtergroups and sort order in the searchCriteriaBuilder

### Fixed Issues (if relevant)

1. magento/magento2#11022: GET v1/products/attribute-sets/sets/list inconsistent return result

### Manual testing scenarios

1. Create an additional product attribute set
1. call to GET http://URL/rest/v1/products/attribute-sets/sets/list?searchCriteria[filter_groups][0][filters][0][field]=attribute_set_name&searchCriteria[filter_groups][0][filters][0][condition_type]=eq&searchCriteria[filter_groups][0][filters][0][value]=default
2. Result should be only the default attribute set (before it always returned all product attribute sets)

### Contribution checklist
